client/router_create.go

```diff
@@ -327,7 +331,13 @@ func (cli *VanClient) GetVanControllerSpec(options types.SiteConfigSpec, van *ty
 				Ports:    controllerPorts,
 				Type:     svctype,
 			},
-		})
+		}
+
+		if options.Controller.LoadBalancerIp != "" {
```

Comment:
Should this also be conditional on the type being lodadbalancer?

Reply:
Technically it shouldn't make a difference, as the field only applies to `LoadBalancer` service types, and others ignore it.
Added the additional check nonetheless.